src/app/ubs/ubs-admin/components/ubs-admin-order-status/ubs-admin-order-status.component.ts

```diff
@@ -37,6 +36,13 @@ export class UbsAdminOrderStatusComponent implements OnChanges, OnInit, OnDestro
     if (changes.currentOrderPrice || changes.totalPaid) {
       this.setOrderPaymentStatus();
     }
+
+    if (changes.generalInfo.currentValue.orderStatus) {
```

Comment:
I think there is a need to have optional chaining here: `changes.generalInfo?.currentValue` but i'm not sure